src/game.cpp

```diff
@@ -7352,7 +7352,7 @@ void game::close( const tripoint &closep )
         }
     } else if( closep == u.pos() ) {
         add_msg(m_info, _("There's some buffoon in the way!"));
-    } else if (m.has_furn(closep) && m.furn_at(closep).close.empty()) {
+    } else if (m.has_furn(closep) && m.furn_at(closep).close) {
```

Comment:
The logic here is inverted. It should be ` && ! furn_at().close` (boolean negation added). Currently it prevents me from closing an open safe with the message "There's a open safe in the way!"